public_html/contact.html

```diff
@@ -137,91 +137,89 @@ <h4 class="panel-title">
 <div class="col-lg-8 col-md-8 mx-auto">
 	<table>
 		<tr>
-			<center>
 				<h3>President</h3>
 				Email: <a class="no-underline" href="mailto:apo-president@wpi.edu" target="blank">apo-president@wpi.edu</a>
-				<p>The President is in charge of supervising the chapter and all other officers. They are at every brother and executive board meeting, represent the chapter at all functions, and communicate with the National Office as needed. They are also responsible for communicating with campus administration and involving chapter advisors in chapter events and matters.</p>
-			</center>
+				<p style="text-align: center;">The President is in charge of supervising the chapter and all other officers. They are at every brother and executive board meeting, represent the chapter at all functions, and communicate with the National Office as needed. They are also responsible for communicating with campus administration and involving chapter advisors in chapter events and matters.</p>
```

Comment:
I think this page looks weird. All the headings and email addresses are left-justified, and the descriptions are centered. They should both be left-justified or both be centered.